src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball firstBall = Lottery.getRandomBall();
+        Ball secondBall = Lottery.getRandomBall();
+        Ball thirdBall = Lottery.getRandomBall();
```

Comment:
The `getRandomBall` method is used statically here, which is not recommended according to the checklist. Instead, create an instance of the Lottery class and use it to call the method.